server/models/ProfileModel.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+  userId: {
+    type: String,
+    required: true,
+  },
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  register_date: {
+    type: Date,
+    default: Date.now,
+  },
+  admin: {
+    type: Boolean,
+    default: false,
+  },
+  address: String,
+  city: String,
+  phone: String,
+  profileImage: String,
+  photos: [],
+  jop: String,
```

Comment:
What is the purpose of this field? :)

Reply:
the phone is needed I saw in the setup-profile-edit.png
the profile image in the notification.png
photos: I think also required as an Array, in case the sitter wants to add a photo as in listing.png
job:  short description for his hoppy like "Dog Care Helper" in the listing.png

Reply:
Thanks for the clarification!